src/styles/utils/_variables.scss

```
@@ -0,0 +1,19 @@
+$cell-size: 100px;
+$cell-gap: 1px;
+$cell-with-gap: $cell-gap + $cell-size;
+$calendar-padding: 10px;
+
+$content-size: 7 * $cell-size + $calendar-padding + 7 * $cell-gap;
```

tell me if I'm mistaken, but I think that to calculte width for a container with 7 days in a row we should include:
- 7 days
- 2 paddings (right and left)
- 6 gaps (there are gaps only BETWEEN elements)

```suggestion
$content-size: 7 * $cell-size + 2 * $calendar-padding + 6 * $cell-gap;
```

---

Yes, I think you're right.